lib/dl_configs/dl_configs/utils.py

```diff
@@ -8,7 +8,7 @@
 from dl_constants.enums import ConnectionType
 
 
-ROOT_CERTIFICATES_FILENAME = "/etc/ssl/certs/ca-certificates.crt"
+ROOT_CERTIFICATES_FILENAME = os.environ.get("REQUESTS_CA_BUNDLE") or "/etc/ssl/certs/ca-certificates.crt"
```

Comment:
no, please, no
we must not read environment variables in common code

Reply:
agreed